gulpfile.js

```diff
@@ -0,0 +1,16 @@
+'use strict';
+
+var gulp = require('gulp');
+var sass = require('gulp-sass');
+
+gulp.task('sass', function() {
+     return gulp.src('./app/**/*.scss')
```

Comment:
We would like to store `.scss` files outside of `app`, please chenge the paths accordingly.